editor/filesystem_dock.cpp

```diff
@@ -3190,16 +3196,10 @@ FileSystemDock::FileSystemDock() {
 	add_child(move_dialog);
 	move_dialog->connect("dir_selected", callable_mp(this, &FileSystemDock::_move_operation_confirm).bind(false));
 
-	rename_dialog = memnew(ConfirmationDialog);
-	VBoxContainer *rename_dialog_vb = memnew(VBoxContainer);
-	rename_dialog->add_child(rename_dialog_vb);
-
-	rename_dialog_text = memnew(LineEdit);
-	rename_dialog_vb->add_margin_child(TTR("Name:"), rename_dialog_text);
-	rename_dialog->set_ok_button_text(TTR("Rename"));
-	add_child(rename_dialog);
-	rename_dialog->register_text_enter(rename_dialog_text);
-	rename_dialog->connect("confirmed", callable_mp(this, &FileSystemDock::_rename_operation_confirm));
+	current_path->set_structured_text_bidi_override(TextServer::STRUCTURED_TEXT_FILE);
+	current_path->set_h_size_flags(SIZE_EXPAND_FILL);
+	_set_current_path_text(path);
+	toolbar_hbc->add_child(current_path);
```

Comment:
Looks like some copy-paste mistake. This shouldn't be here at all.

Reply:
You are totally correct. I removed the modification.